src/arm/64/mc.S

```diff
@@ -2329,6 +2329,7 @@ endfunc
 filter_fn put,  x0, x1, x2, x3, w4, w5, w6, x6, w7, x7, x8, x9, 10
 filter_fn prep, x0, x7, x1, x2, w3, w4, w5, x5, w6, x6, x8, x9, 6
 
+#if 0
```

Comment:
Now that #1859 is merged and the `mc_warp_filter` table is available, I think we can skip this patch: "arm64: mc: Don't compile warp8x8{,t} for now"